src/pipx/main.py

```diff
@@ -1087,6 +1100,9 @@ def setup(args: argparse.Namespace) -> None:
         print_version()
         sys.exit(0)
 
+    if not constants.WINDOWS and args.is_global:
```

Comment:
I don’t remember the context… why is Windows excluded here?

Reply:
I don't know the background for that decision, but its explicitly mentioned in the docs that the `--global` flag is not available on Windows.

Quoting from https://pipx.pypa.io/stable/how-pipx-works/:
> * adding --global flag to any pipx command will execute the action in global scope which will expose app to all users - [reference](https://pipx.pypa.io/stable/installation/#global-installation). **Note that this is not available on Windows.**

Reply:
It's because the submitter did not have access to a Windows system and there was no consensus (or even a discussion) about file system locations on Windows that `pipx` acting globally should install to. Once someone needing this feature steps up, we'll be happy to take it further.